vignettes/terminal.Rmd

```diff
@@ -0,0 +1,100 @@
+---
+title: "Terminal Interaction"
```

Comment:
Can we use `Interacting with Terminals` as the title here? (For symmetry with the other titles which also are of the form `Interacting with <...>`)